From b4c01804de12d899b659374e5128cce1cf32905e Mon Sep 17 00:00:00 2001
From: Boleslaw Ciesielski <combolek@users.noreply.github.com>
Date: Fri, 8 Jan 2021 10:10:00 -0800
Subject: [PATCH] Fixed build regression in commit
 a5882a1a1c3f561233a22cc7663514b97e5fb3ba on some Linux platforms. We need to
 link with GTK libs after all.

---
 src/slic3r/CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/slic3r/CMakeLists.txt b/src/slic3r/CMakeLists.txt
index 019850a98e..9568a77226 100644
--- a/src/slic3r/CMakeLists.txt
+++ b/src/slic3r/CMakeLists.txt
@@ -258,9 +258,10 @@ endif ()
 
 # We need to implement some hacks for wxWidgets and touch the underlying GTK
 # layer and sub-libraries. This forces us to use the include locations of these
-# libraries. No need to link to them, wxWidgets does that already.
-# See PresetComboBox.cpp for the includes and subsequent workarounds.
+# layer and sub-libraries. This forces us to use the include locations and
+# link these libraries.
 if (UNIX AND NOT APPLE)
     find_package(GTK${SLIC3R_GTK} REQUIRED)
     target_include_directories(libslic3r_gui PRIVATE ${GTK${SLIC3R_GTK}_INCLUDE_DIRS})
+    target_link_libraries(libslic3r_gui ${GTK${SLIC3R_GTK}_LIBRARIES})
 endif ()
